app/src/views/private/components/files-navigation.vue

```diff
@@ -62,32 +65,54 @@
 </template>
 
 <script setup lang="ts">
-import { Folder, useFolders } from '@/composables/use-folders';
-import { isEqual } from 'lodash';
-import { watch } from 'vue';
 import { useI18n } from 'vue-i18n';
-import NavigationFolder from './navigation-folder.vue';
+import { ref, watch } from 'vue';
+import { useFolders, openFoldersInitial } from '@/composables/use-folders';
+import NavigationFolder from './files-navigation-folder.vue';
```

Comment:
```suggestion
import NavigationFolder from '@/views/private/components/files-navigation-folder.vue';
```

Should this be updated for consistency?